src/App.js

```diff
@@ -1,27 +1,35 @@
 import React from "react";
 import { Route, Routes } from "react-router-dom"
-import Header from "./Header/Header";
-import Footer from "./components/Footer/Footer";
-import './App.scss';
+
+import './App.scss'
+
+//componets
+import Header from "./Components/Header/Header";
```

Comment:
I'd suggest to have componets folder written with small letter `./components/....` rename it in your local project